build/Build.Pack.cs

```diff
@@ -520,7 +521,16 @@ string ConstructRedHatPackageFilename(string packageName, string architecture)
             const string description = "The deployment agent that is installed on each machine you plan to deploy to using Octopus.";
             const string author = "Octopus Deploy";
             const string title = "Octopus Tentacle cross platform bundle";
-            OctoCliTool($@"pack --id=Octopus.Tentacle.CrossPlatformBundle --version={FullSemVer} --basePath={workingDirectory} --outFolder={ArtifactsDirectory / "nuget"} --author=""{author}"" --title=""{title}"" --description=""{description}""");
+
+            OctopusTasks.OctopusPack(o => o
```

Comment:
This is really nice! How did you find it and know to use it?

Reply:
While trying to understand what was going on in the argument parsing (it was having issues with quotes in arguments I downloaded the Nuke source and discovered this while browsing.

Reply:
noice